docs/dpdocsutils/dpdocsutils/previews.py

```diff
@@ -36,7 +36,7 @@ def app_to_image(app_path, image_path, width, height):
     )
 
 
-def embed_local_app(app_path, width, height, iframe=False):
+def embed_local_app(app_path, width, height, iframe=True):
     if iframe:
         return IFrame(app_path, width=width, height=height)
```

Comment:
I wonder if we can set `loading="lazy"` here to defer fetching..

Reply:
Running a build locally, you add it in using `extras=['loading="lazy"'].

```suggestion
        return IFrame(app_path, width=width, height=height, extras=['loading="lazy"'])
```

Reply:
Works a treat on Chrome, not supported on Firefox.

I think we want this, @shahinrostami.
Have a play with scrolling through the Accessing Data in Python guide. Causes more caching to happen, too (requests aren't racing each other anymore).

Reply:
Nice, have commit the suggestion. It didn't work in Safari either, unfortunately.